packages/editor-ui/src/components/ExecutionsView/ExecutionPreview.vue

```diff
@@ -33,21 +33,22 @@
 				</n8n-text>
 			</div>
 			<div>
-				<el-dropdown v-if="executionUIDetails.name === 'error'" trigger="click" class="mr-xs" @command="handleRetryClick">
+				<el-dropdown v-if="executionUIDetails.name === 'error'" trigger="click" class="mr-xs" @command="handleRetryClick" ref="retryDropdown">
 					<span class="retry-button">
 						<n8n-icon-button
 							size="large"
 							type="tertiary"
 							:title="$locale.baseText('executionsList.retryExecution')"
 							icon="redo"
+							@blur="onRetryButtonBlur"
 						/>
 					</span>
 					<el-dropdown-menu slot="dropdown">
 						<el-dropdown-item command="current-workflow">
 							{{ $locale.baseText('executionsList.retryWithCurrentlySavedWorkflow') }}
 						</el-dropdown-item>
 						<el-dropdown-item command="original-workflow">
-							{{ $locale.baseText('executionsList.retryWithOriginalworkflow') }}
+							{{ $locale.baseText('executionsList.retryWithOriginalWorkflow') }}
```

Comment:
TS should have prevented the typo. Do you know how this made it through?

Reply:
Well, there was a TS error but looks like it was ignored...
Just looking at the file history, seems like it was there all along :/

Reply:
Looks like Vetur only highlights type issues in `<script>` but not in `<template>`. Apparently no longer an issue with Volar.